tests/test_ui.py

```diff
@@ -179,6 +196,24 @@ def addSample(self):
         widget_2 = self.getDockedWidget(self.window.docks, InsertPrimitiveDialog.dock_flag)
         self.assertIsNot(widget, widget_2)
         old_vertex_count = len(self.model.sample.vertices)
+
+        self.window.sample_properties_dialog_action.trigger()
```

Comment:
The dialog is already open. You don need to trigger the action again or assert the instance

Reply:
Removed the redundant assert statement and the unnecessary widget trigger.